src/serverassert.h

```diff
@@ -38,7 +38,23 @@
 #ifndef VALKEY_ASSERT_H
 #define VALKEY_ASSERT_H
 
-#include "config.h"
+/* This is a drop-in replacement of assert. We use it even in code copied from
+ * other sources. To make these files usable outside of Valkey, it should be
+ * enough to replace this include with <assert.h>. Therefore, this file
+ * shouldn't have any dependencies to any other valkey code. */
```

Comment:
```suggestion
/* Since this file is a drop-in replacement of assert.h, this file shouldn't have any dependencies 
 * to any other Valkey code. */
```

This comment is already kind of at the top of the file, so maybe simplify it a bit.

Reply:
Thanks, I didn't see that!

Reply:
```suggestion
/* This file shouldn't have any dependencies to any other Valkey code. */
```